### DISCUSSION TITLE
How to install different version than in Repo

### DISCUSSION THREAD

**Discussion by redboyhun** [procount/pinn#644] [Q&A] [answered]
Hi,
I looked at the OS list and I see both OSs (Batocera and LibreElec), but I'd like to install different versions.
How can I do it?
I'd like LibreElec v9 (Kodi v18.9, offered v19) and Batocera v33 (offered v30).
For both images I can only download img.gz and no tar.xz.

Thank you for the help

FYI I have RPi 2

**Comment by procount** [comment]
Each OS needs to be converted before it can be installed by PINN (to make it multi-bootable).
LibreELEC is updated by its maintainers for NOOBS, so I just present whatever is their latest version.
Batocera is converted by @matthuisman - Matt, are you able to update this to v33?

**Comment by matthuisman** [comment]
my source uses the tarballs provided by batocera. i dont create any.
So whatever version he published, should be used.
the v30 just comes from our os.json - but it may not be correct

**Comment by redboyhun** [comment]
to install different versions from USB, I can't simply extract the IMG file and pack the content of "0.fat" as .tar right?
![image](https://user-images.githubusercontent.com/34648596/160570959-c4818679-5d75-4d52-b892-f889c0740429.png)

**Comment by procount** [comment]
@matthuisman - where do you get the tarballs from? I can only find images.

**Comment by matthuisman** [comment, marked as answer]
https://github.com/matthuisman/pinn-os/blob/7de09f744bb4d9d54f063f821740df395bff9e44/os_list_v3.json#L166

**Comment by redboyhun** [comment]
I could find the LibreElec packages here: https://ftp.fau.de/libreelec/
LibreELEC-RPi2.arm-9.2.8-noobs.tar
LibreELEC-RPi2.arm-9.2.8.img.gz
LibreELEC-RPi2.arm-9.2.8.tar

I guess the *-noobs.tar would be the best option to install from USB, right?

**Reply by matthuisman** [reply]
Good find. Yes, the noobs.tar should have basically all you need

**Comment by frozen-cdn** [comment]
Hi friends, is there any way to get the new Android TV 13 version that was just released ?  Thank you!

https://www.xda-developers.com/raspberry-pi-4-android-tv-13-custom-rom/
